tools/gcp/utils/keep_xds_interop_resources.json

```diff
@@ -0,0 +1,7 @@
+{
+    "gce_framework": {
+        "suffix": [
+            "1620244457-menghanl"
```

Comment:
Is this a real leaked resource?

Reply:
Yes, it's a real leaked instance template, and there is no TD resource associated with it. I will remove it before submission.